README.md

```diff
@@ -11,7 +11,6 @@
   [asking]: https://stackoverflow.com/questions/ask?tags=sbt
   [LICENSE]: LICENSE
   [sbt/io]: https://github.com/sbt/io
-  [sbt/util]: https://github.com/sbt/util
```

Comment:
I started hacking over there after seeing this, only to realize that it had been inlined, so I guess it's better to remove it from here? Or maybe you are planning to cherry-pick stuff anyway?

Reply:
This was an oversight. Thanks for the change.